src/components/apps/launch/AppInfo.js

```diff
@@ -105,25 +111,84 @@ const UnavailableMsg = ({ app, hasDeprecatedParams, baseId }) => {
 };
 const AppInfo = React.forwardRef((props, ref) => {
     const { app, baseId, hasDeprecatedParams, loading, loadingError } = props;
+    const { t } = useTranslation("common");
+    const { t: i18nApps } = useTranslation("apps");
+    const router = useRouter();
     const classes = useStyles();
     const theme = useTheme();
     const isMobile = useMediaQuery(theme.breakpoints.down("xs"));
+    const [detailsDrawerOpen, setDetailsDrawerOpen] = React.useState(false);
+    const [docDialogOpen, setDocDialogOpen] = React.useState(false);
     return (
         <div ref={ref}>
+            <Button
+                color="primary"
+                variant={isMobile ? "text" : "contained"}
+                size="small"
+                style={{
+                    margin: !isMobile
+                        ? theme.spacing(0.5)
+                        : theme.spacing(0),
```

Comment:
This could also be updated for consistency:
```suggestion
                    margin: isMobile
                        ? theme.spacing(0)
                        : theme.spacing(0.5),
```